src/main/flight/filter.h

```diff
@@ -0,0 +1,11 @@
+/*
+ * filter.h
+ *
+ *  Created on: 24 jun. 2015
+ *      Author: borisb
+ */
+
+
+typedef float filterState_t;
```

Comment:
this type is filter specific - other filters may have different storage requirements. So it would be better to include filter name in this type